.github/workflows/deploy-pre-release.yml

```diff
@@ -0,0 +1,128 @@
+---
+name: Deploy Pre-Release Artifacts
+
+on:
+  push:
+    branches:
+      - develop
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  LANG: en_US.utf-8
+  LC_ALL: en_US.utf-8
+  PYTHON_VERSION: '3.8'
+  PROJECT_NAME: test-release-actions
+  RUNS_ON: ubuntu-20.04
+
+jobs:
+
+  bump_version:
+    environment: test_env
+    runs-on: ubuntu-20.04
+    # Only run this job if the commit message doesn't start with release sentinel.
```

Comment:
For contributors, maybe in the README we should document that "Release" has special meaning when creating a commit message.